interface/billing/sl_eob_search.php

```diff
@@ -528,7 +526,14 @@ function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID =
                 continue;
             }
         } else {
-            fwrite($fhprint, make_statement($stmt));
+            if ($inv_pid[$inv_count] != $inv_pid[$inv_count+1]) {
+                $tmp = make_statement($stmt);
+                if(empty($tmp)) {
+                    $tmp = xlt("This EOB Id: $inv_pid[$inv_count] Encounter: $stmt[encounter] does not meet minumum print requirements setup in Globals or there is an unknown error."."\n");
```

Comment:
sure, just use the translation comments mechanism :)
``````
xl("y'all can't do that{{Jerry dialect for You can not do that}}");